site/profile/manifests/core/hardware.pp

```diff
@@ -3,7 +3,16 @@
 #   For *baremetal* only.
 #
 class profile::core::hardware {
-  include profile::core::powertop
+  case fact('dmi.manufacturer') {
+    /Advantech/, /VersLogic Corp\./: {
```

Comment:
This isn't the right pattern, it should match eg "VersaLogic Corporation".

Reply:
They seem to be confused about their own name: 
```
[root@comcam-lion01 ~]# dmidecode -t 1
# dmidecode 3.2
Getting SMBIOS data from sysfs.
SMBIOS 3.0 present.

Handle 0x000B, DMI type 1, 27 bytes
System Information
	Manufacturer: VersaLogic Corporation
	Product Name: VersaLogic Corporation
	Version: 0.1
	Serial Number: System Serial Number
	UUID: 5c4606fa-192f-453a-b299-7b088c63bb9b
	Wake-up Type: Other
	SKU Number: System SKUNumber
	Family: Kabylake System

[root@comcam-lion01 ~]# dmidecode -t 2
# dmidecode 3.2
Getting SMBIOS data from sysfs.
SMBIOS 3.0 present.

Handle 0x000C, DMI type 2, 15 bytes
Base Board Information
	Manufacturer: VersLogic Corp.
	Product Name: EPMe-42
	Version: 2
	Serial Number: 1
	Asset Tag: Base Board Asset Tag
	Features:
		Board is a hosting board
		Board is replaceable
	Location In Chassis: Part Component
	Chassis Handle: 0x0000
	Type: Motherboard
	Contained Object Handles: 0

```

Reply:
And different boards seem to have different values:
```
[root@comcam-lion03 ~]# dmidecode -t 1
# dmidecode 3.2
Getting SMBIOS data from sysfs.
SMBIOS 3.0.0 present.

Handle 0x000B, DMI type 1, 27 bytes
System Information
	Manufacturer: VersaLogic Corporation
	Product Name: EPMe-42
	Version: 0.1
	Serial Number: System Serial Number
	UUID: 5c4606fa-192f-453a-b299-7b088c63bb9b
	Wake-up Type: Other
	SKU Number: System SKUNumber
	Family: Kabylake System

[root@comcam-lion03 ~]# dmidecode -t 2
# dmidecode 3.2
Getting SMBIOS data from sysfs.
SMBIOS 3.0.0 present.

Handle 0x000C, DMI type 2, 15 bytes
Base Board Information
	Manufacturer: VersaLogic Corp.
	Product Name: EPMe-42
	Version: 2
	Serial Number: 1
	Asset Tag: Base Board Asset Tag
	Features:
		Board is a hosting board
		Board is replaceable
	Location In Chassis: Part Component
	Chassis Handle: 0x0000
	Type: Motherboard
	Contained Object Handles: 0
```

Reply:
lets try again.